ui-participant/src/hub/documents/DocumentLibrary.tsx

```diff
@@ -20,10 +27,20 @@ export default function DocumentLibrary() {
     pStudy.study.studyEnvironments.find(studyEnv =>
       studyEnv.environmentName === portalEnv.environmentName))?.study
 
+  if (!currentStudy) {
+    return <div>Study not found</div>
+  }
```

Comment:
I think returning early before a `useEffect`/`useState` can make react mad (but maybe it's ok in this instance becuase currentStudy isn't likely to go from null->not null, I'm not sure)